src/discordtipbot/discordbot.cr

```diff
@@ -152,6 +152,14 @@ class DiscordBot
         end
       end
     end
+
+    # Make use of the status to display info
+    spawn do
+      sleep 2
```

Comment:
?

Reply:
If immediately used, the discord bot isn't authenticated yet, and discord closes the connection

Reply:
This fiber should be spawned from READY then.

Reply:
Good idea 🤔